### ISSUE TITLE
Is there any chance to install this in AWS lambda ?

### ISSUE THREAD

**Issue by adantart** [deanmalmgren/textract#317]
I mean, my question is, which libraries and files are necessary to zip them for AWS Lambda purposes ?

**Comment by jpweytjens** [comment]
It depends on which filetype you want to parse. Textract is just a wrapper for external parsers. Most are python packages, but textract uses external CLI tools as well.

**Comment by skjortan23** [comment]
I think what the OP: means is that the resulting zip file when installing all dependencies is> 55M.
This means that it is not possible to run on AWS lambda. 

This is a showstopper for me as well. 

`
 An error occurred (RequestEntityTooLargeException) when calling the 
 CreateFunction operation: Request must be smaller than 69905067 bytes for the 
 CreateFunction operation

This is likely because the deployment package is 54.9 MB. Lambda only allows 
deployment packages that are 50.0 MB or less in size. To avoid this error, 
decrease the size of your chalice application by removing code or removing 
dependencies from your chalice application.
`

**Comment by jpweytjens** [comment]
I have no experience with AWS Lambda, so thank you for pointing that out.

Most of the dependencies are very small, but SpeechRecognition takes up ~32MB. I'll think about making some dependencies optional to reduce filesize.

**Comment by skjortan23** [comment]
@jpweytjens yes to work around this I made a fork yesterday and just commented out the dependency on SpeechRecognition and successfully deployed to lambda.
